src/components/displayDocuments/documents.tsx

```diff
@@ -2,38 +2,47 @@ export interface Document {
   title: string;
   url: string;
   imageName: string;
+  tags: Tag[];
 }
 
+type Tag = "Tradetrust" | "OpenCerts" | "Licence";
```

Comment:
Will prefer over enum over type here. The transpiled version will still be string.

https://stackoverflow.com/questions/49761972/difference-between-string-enums-and-string-literal-types-in-ts

Reply:
done